catalog/internal/utils.go

```diff
@@ -284,12 +284,10 @@ func CreateViewMetadata(
 		"view-uuid":          viewUUID,
 		"format-version":     1,
 		"location":           loc,
-		"schema":             schema,
+		"schemas":            []*iceberg.Schema{schema},
 		"current-version-id": versionId,
-		"versions": map[string]interface{}{
-			"1": viewVersion,
-		},
-		"properties": props,
+		"versions":           []interface{}{viewVersion},
```

Comment:
at this rate we might as well properly create the version type with the appropriate required/optional values etc.